app/lib/report_helper.rb

```diff
@@ -61,6 +61,9 @@ def get_raw_taxon_counts(report)
       taxon_counts.count               AS  count,
       (count / #{total_reads}.0
         * 1000000.0)                   AS  rpm,
+      taxon_counts.percent_identity    AS  percent_identity,
```

Comment:
What are the default values for these columns?   If they can be NULL, then I'd prefer suitable defaults be set here in the query --- unless it can be done in the DB schema.   In the query you can use IF syntax like for IF(stddev IS NULL, ...) below.

Reply:
If they are not set, then no value should be shown at all (as is currently the case for all the samples that haven't been reprocessed with the modified pipeline). There is really no suitable default value for the case where you have a hit but no percent-identity/alignment-length/e-value

Reply:
it should be fine. NULL shouldn't really happen but if it does, showing blank should be ok. 
Maybe add a TODO: comment if we plan to filter on this field in the future.

Reply:
Got it, we'll need to set defaults when we implement filtering on these fields. I've added a "to do" comment.